### ISSUE TITLE
how to convert obstacle.pt to onnx 

### ISSUE THREAD

**Issue by duanchengwen** [ApolloAuto/apollo#15086]
We appreciate you go through Apollo documentations and search previous issues before creating an new one. If neither of the sources helped you with your issues, please report the issue using the following form. Please note missing info can delay the response time.

### System information

- **OS Platform and Distribution (e.g., Linux Ubuntu 18.04)**:
- **Apollo installed from (source or binary)**:
- **Apollo version (3.5, 5.0, 5.5, 6.0)**: 7.0
- **Output of `apollo.sh config` if on `master` branch**:

### Steps to reproduce the issue:
- **Please use bullet points and include as much details as possible**:

### Supporting materials (screenshots, command lines, code/script snippets):

how to convert obstacle.pt to onnx？
I want to use the onnx model to launch dag_streaming_perception_camera.dag, what code do I need to change？

**Comment by daohu527** [comment]
No, this is already a model file exported by pytorch, I am not sure whether it can be converted to onnx. The one you choose is probably the `SMOKE` model, you can find a version online that can be converted to onnx

**Comment by LordonCN** [comment]
You can load it without training code only if it is 'ckpt file', so try to use torch.load function to load obstacle.pt file. If the loading is successful, then you can export onnx model by your self(torch.onnx._export())
